client/src/assets/scss/main.scss

```diff
@@ -8,6 +8,10 @@
   box-sizing: border-box;
 }
 
+body {
+  background-color: #FFEBA1; 
```

Comment:
you might consider creating color variables to use throughout your styling in multiple places
https://sass-lang.com/documentation/variables

```css
$white: #FFFFFF;

body {
  color: $white;
}
```
not exactly what you're doing, but an example of how to use color variables

that way if you update colors later, you only need to change the hex/rgb code assigned to the color variable rather than updating multiple color codes!